truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/TruffleLanguage.java

```diff
@@ -437,6 +439,13 @@ public OutputStream err() {
         public Instrumenter instrumenter() {
             return instrumenter;
         }
+
+        /**
+         * @return arguments used to create the polyglot engine
+         */
+        public Map<String, Map<String, Object>> getConfig() {
+            return config;
```

Comment:
This should return Map<String,Object>. Javadoc should connect this method with the config one. See
https://github.com/jtulach/truffle/blob/32d4240880fd2a97100c3eb1776e274d7c8bbfbb/truffle/com.oracle.truffle.api/src/com/oracle/truffle/api/TruffleLanguage.java#L445